packages/design-tokens/src/tokens/functional/components/image/base.json

```diff
@@ -0,0 +1,11 @@
+{
+  "brand": {
+    "Image": {
+      "ratio": {
+        "custom": {
+          "value": "3 / 2"
```

Comment:
Could this be `none` to avoid being opinionated about this by default?

Reply:
I believe that auto is the default so I will set it to that. If that works for you.